### ISSUE TITLE
Cabal can't find libtinfo when libtinfo5 is installed

### ISSUE THREAD

**Issue by vmchale** [clash-lang/clash-lang.github.io#3]
I am using linux mint. The file libtinfo.so is not present but rather the file libtinfo.so.5. Creating a symlink to libtinfo.so.5 named libtinfo.so fixes the issue but otherwise the build fails. 

This is the error message:
$ cabal install clash-ghc --enable-documentation --enable-executable-dynamic
Warning: The package list for 'hackage.haskell.org' is 22.5 days old.
Run 'cabal update' to get the latest list of available packages.
Resolving dependencies...
Configuring clash-ghc-0.6.21...
Building clash-ghc-0.6.21...
Preprocessing executable 'clash' for clash-ghc-0.6.21...
[ 1 of 12] Compiling CLaSH.GHC.GHC2Core ( src-ghc/CLaSH/GHC/GHC2Core.hs, dist/dist-sandbox-6100cc0b/build/clash/clash-tmp/CLaSH/GHC/GHC2Core.dyn_o )
<command line>: can't load .so/.DLL for: libtinfo.so (libtinfo.so: cannot open shared object file: No such file or directory)
Failed to install clash-ghc-0.6.21
cabal: Error: some packages failed to install:
clash-ghc-0.6.21 failed during the building phase. The exception was:
ExitFailure 1

**Comment by christiaanb** [comment]
Thanks for the report. I'm also running linux mint, and it might well be that I symlinked libtinfo without giving it too much thought. I'll check it out when I get back in the office tomorrow, and then update the installation instructions accordingly.

**Comment by christiaanb** [comment]
I indeed have a symlink from `/usr/lib/x86_64-linux-gnu/libtinfo.so` to `/lib/x86_64-linux-gnu/libtinfo.5.so` on my machine. Sadly, I can't figure out who created it, whether I did it or `apt-get` did it.

Regardless, I'll update the installation instructions. 

@vmchale One question: Given that you are building with `--enable-executable-dynamic`, are you building for a 32-bit platform? or was there some other reason you are building with that flag?
